CONTRIBUTING.md

```diff
@@ -6,9 +6,12 @@ Work done surrounding this project is governed by the
 
 ## Dependencies
 The `requirements.txt` file is frozen to the currently tested
-versions of `requirements.base`. To update, create a venv,
-run `python3 -m pip install -r requirements.base`, run all tests,
-and then update `requirements.txt` with the output from `pip freeze`.
+versions of `requirements.in`. To update, create a venv,
+install [`pip-tools`](https://pypi.org/project/pip-tools)
+run `python3 -m pip-sync`, run all tests, and then update 
+`requirements.txt` with `python3 -m pip-compile --upgrade` command.
```

Comment:
Tested locally, the command `python3 -m pip-compile --upgrade` doesn't seem to work. I had to do just `pip-compile --upgrade`.